packages/utils/package-manager.ts

```diff
@@ -13,17 +13,13 @@ import * as path from "path";
  */
 
 export function getPackageManager(): string {
-	const hasLocalNPM = fs.existsSync(path.resolve(process.cwd(), "package-lock.json"));
 	const hasLocalYarn = fs.existsSync(path.resolve(process.cwd(), "yarn.lock"));
-	if (hasLocalNPM) {
-		return "npm";
-	} else if (hasLocalYarn) {
-		return "yarn";
-	} else if (spawn.sync("yarn", [" --version"], { stdio: "ignore" }).error) {
-		return "npm";
-	} else {
-		return "npm";
+	let defaultpackageManager = "npm";
```

Comment:
The logic is basically the same. What if `yarn` is installed but the user hasn't done a first installation yet? Checking if it is installed means trying to execute `yarn` command, I think